osu.Game/Database/ArchiveModelManager.cs

```diff
@@ -452,41 +447,6 @@ void rollback()
             return item.ToEntityFrameworkLive();
         }, cancellationToken, TaskCreationOptions.HideScheduler, lowPriority ? import_scheduler_low_priority : import_scheduler).Unwrap().ConfigureAwait(false);
 
-        /// <summary>
-        /// Exports an item to a legacy (.zip based) package.
-        /// </summary>
-        /// <param name="item">The item to export.</param>
-        public void Export(TModel item)
-        {
-            var retrievedItem = ModelStore.ConsumableItems.FirstOrDefault(s => s.ID == item.ID);
-
-            if (retrievedItem == null)
-                throw new ArgumentException(@"Specified model could not be found", nameof(item));
```

Comment:
Note that this has been removed from the version in this PR, but I couldn't seem to break this, so I'm not sure it's still required. At least after switching to realm one would hope it is not.